src/MSALWrapper.Test/AuthFlow/IntegratedWindowsAuthenticationTest.cs

```diff
@@ -81,36 +86,33 @@ public async Task GetCachedToken_ReturnsNull()
         {
             this.MockAccount();
             this.CachedAuthReturnsNull();
+            this.IWAReturnsResult();
 
             // Act
             AuthFlow.IntegratedWindowsAuthentication iwa = this.Subject();
             var authFlowResult = await iwa.GetTokenAsync();
 
             // Assert
-            this.pcaWrapperMock.VerifyAll();
-            authFlowResult.TokenResult.Should().Be(null);
-            authFlowResult.Errors.Should().HaveCount(1);
+            authFlowResult.TokenResult.Should().Be(this.tokenResult);
+            authFlowResult.Errors.Should().BeEmpty();

```

Comment:
We should also assert the following:
`authFlowResult.TokenResult.IsSilent.Should().BeTrue();`